src/gui/footer/footer_line.cpp

```diff
@@ -41,6 +41,9 @@ bool FooterLine::Create(footer::Item item_id, size_t index) {
 
     if (item_id >= footer::Item::_count) {
         item_id = footer::Item::none;
+    } else if (item_id == footer::Item::none) {
+        unregister(index);
+        return false;
```

Comment:
It is rather debatable whether this should call `true` or `false`. From the previous code, it rather seems like the function should return `true` if an item is changed (and in that case, and only in that case, it should call reposition). This code does not follow that logic.

But looking around, it seems like the return value is not used anywhere, so it would perhaps be best to just turn it into a procedure.